Source/WebCore/platform/graphics/GraphicsLayer.h

```diff
@@ -530,6 +531,8 @@ class GraphicsLayer : public RefCounted<GraphicsLayer> {
 #endif
     virtual bool usesContentsLayer() const { return false; }
 
+    virtual void setRemoteFrame(RemoteFrame&) { }
```

Comment:
This is a layering violation. RemoteFrame is a `page` concept, and here we're in `platform` code.